components/Footer.js

```diff
@@ -65,7 +65,7 @@ const StyledFooter = styled.footer`
   padding: 2em;
   hr {
     border-width: 0.4px;
-    width: ${props => props.theme.large};
+    max-width: ${props => props.theme.large};
```

Comment:
This `<hr>` having a `width` larger than the viewport was messing with the `window.innerWidth` value we were getting back on mobile, forcing the filter panel open by default.

Reply:
That's an interesting effect!